tests/phpunit/tests/link/getPreviousCommentsLink.php

```diff
@@ -35,4 +35,16 @@ public function test_page_should_default_to_1_when_no_cpage_query_var_is_found()
 
 		set_query_var( 'cpage', $cpage );
 	}
+
+	/**
+	 * @ticket 60806
+	 */
+	public function test_page_should_respect_value_of_page_argument() {
+		$p = self::factory()->post->create();
+		$this->go_to( get_permalink( $p ) );
+
+		$link = get_previous_comments_link( 'Next', 3 );
```

Comment:
While it doesn't affect the tests, this is a bit misleading, so let's change it to

```suggestion
		$link = get_previous_comments_link( 'Previous', 3 );
```

Reply:
You're right. I copied it from the other tests in that file and didn't realize. I can change those as well even if they are not related to this PR.

Reply:
Done in [this commit](https://github.com/WordPress/wordpress-develop/pull/7275/commits/42fd604d0ac32352cf29aaaaed1c5a42a0b46b14).